log/std.go

```diff
@@ -0,0 +1,177 @@
+	l.outputf(SeverityInfo, format, a...)
+}
+
+// Infoln logs to the INFO log.
+// Arguments are handled in the manner of fmt.Println; a newline is appended if missing.
+func (l *stdLogger) Infoln(a ...interface{}) {
+	l.outputln(SeverityInfo, a...)
+}
+
+// Warning logs to the WARNING logs.
+// Arguments are handled in the manner of fmt.Print; a newline is appended if missing.
+func (l *stdLogger) Warning(a ...interface{}) {
+	l.output(SeverityWarning, a...)
+}
+
+// Warningf logs to the WARNING logs.
+// Arguments are handled in the manner of fmt.Printf; a newline is appended if missing.
+func (l *stdLogger) Warningf(format string, a ...interface{}) {
+	l.outputf(SeverityWarning, format, a...)
+}
+
+// Warningln logs to the WARNING logs.
+// Arguments are handled in the manner of fmt.Println; a newline is appended if missing.
+func (l *stdLogger) Warningln(a ...interface{}) {
+	l.outputln(SeverityWarning, a...)
+}
+
+// Error logs to the ERROR logs.
+// Arguments are handled in the manner of fmt.Print; a newline is appended if missing.
+func (l *stdLogger) Error(a ...interface{}) {
+	l.output(SeverityError, a...)
+}
+
+// Errorf logs to the ERROR logs.
+// Arguments are handled in the manner of fmt.Printf; a newline is appended if missing.
+func (l *stdLogger) Errorf(format string, a ...interface{}) {
+	l.outputf(SeverityError, format, a...)
+}
+
+// Errorln logs to the ERROR logs.
+// Arguments are handled in the manner of fmt.Println; a newline is appended if missing.
+func (l *stdLogger) Errorln(a ...interface{}) {
+	l.outputln(SeverityError, a...)
+}
+
+// Fatal logs to the FATAL logs, then calls os.Exit(1).
+// Arguments are handled in the manner of fmt.Print; a newline is appended if missing.
+func (l *stdLogger) Fatal(a ...interface{}) {
+	l.output(SeverityFatal, a...)
+}
+
+// Fatalf logs to the FATAL logs, then calls os.Exit(1).
+// Arguments are handled in the manner of fmt.Printf; a newline is appended if missing.
+func (l *stdLogger) Fatalf(format string, a ...interface{}) {
+	l.outputf(SeverityFatal, format, a...)
+}
+
+// Fatalln logs to the FATAL logs, then calls os.Exit(1).
+// Arguments are handled in the manner of fmt.Println; a newline is appended if missing.
+func (l *stdLogger) Fatalln(a ...interface{}) {
+	l.outputln(SeverityFatal, a...)
+}
+
+func (l *stdLogger) output(severity Severity, a ...interface{}) {
+	l.write(prefix(severity, l.wrapper+2) + fmt.Sprint(a...))
+	l.exitIfFatal(severity)
+}
+
+func (l *stdLogger) outputf(severity Severity, format string, a ...interface{}) {
+	l.write(prefix(severity, l.wrapper+2) + fmt.Sprintf(format, a...))
+	l.exitIfFatal(severity)
+}
+
+func (l *stdLogger) outputln(severity Severity, a ...interface{}) {
+	l.write(prefix(severity, l.wrapper+2) + fmt.Sprintln(a...))
+	l.exitIfFatal(severity)
+}
+
+func (l *stdLogger) exitIfFatal(severity Severity) {
+	if severity == SeverityFatal {
+		os.Exit(1)
+	}
+}
+
+func (l *stdLogger) write(data string) {
+	if data[len(data)-1] != '\n' {
```

Comment:
what if i logged empty string?

Reply:
It's impossible.  `data` has `header` at least.
```
INFO  1222-18:01:16.566+08 log_test.go:94 | 
```

Reply:
It's impossible.  `data` has `header` at least.
```
INFO  1222-18:01:16.566+08 log_test.go:94 | 
```